src/client/app/utils/helperFunctions.js

```diff
@@ -16,7 +16,7 @@ const debounce = (func, wait, immediate) => {
         timeout = setTimeout(later, wait);
 
         if (callNow) {
-            func.apply(this, args);
+            func.apply(this, params);
```

Comment:
I have no better idea for achieving the end result, but am totally open to refactoring